nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -166,317 +108,158 @@
             .identifiesControllerService(SSLContextService.class)
             .build();
 
-    public static final PropertyDescriptor CLIENT_AUTH = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor CLIENT_AUTH = new PropertyDescriptor.Builder()
             .name("CLIENT_AUTH")
             .displayName("Client Auth")
             .description("The client authentication policy to use for the SSL Context. Only used if an SSL Context Service is provided.")
             .required(false)
             .allowableValues(SSLContextService.ClientAuth.NONE.toString(), SSLContextService.ClientAuth.REQUIRED.toString())
             .build();
 
-    @Override
```

Comment:
I believe you are talking about customValidate() you had there before. I think it was an oversight on my end. Will put it back